src/components/assets/Account.vue

```diff
@@ -0,0 +1,236 @@
+
+      <div class="row">
+        <span>{{ $t('assets.totalBalance') }}</span>
+        <span class="text--total-balance"> ${{ $n(balUsd + ttlErc20Amount) }} </span>
+      </div>
+    </div>
+  </div>
+</template>
+<script lang="ts">
+import { defineComponent, ref, watchEffect, computed, watch } from 'vue';
+import {
+  setAddressMapping,
+  getEvmMappedSs58Address,
+  getShortenAddress,
+} from 'src/hooks/helper/addressUtils';
+import { SupportWallet, supportWalletObj } from 'src/config/wallets';
+import { useStore } from 'src/store';
+import { getSelectedAccount } from 'src/hooks/helper/wallet';
+import { useAccount, useBalance, useBreakpoints, useConnectWallet, usePrice } from 'src/hooks';
+import { endpointKey, getProviderIndex, providerEndpoints } from 'src/config/chainEndpoints';
+import { ethers } from 'ethers';
+import { $api } from 'src/boot/api';
+import { isValidEvmAddress } from 'src/config/web3';
+import { useMetamask } from 'src/hooks/custom-signature/useMetamask';
+
+export default defineComponent({
+  props: {
+    ttlErc20Amount: {
+      type: Number,
+      required: true,
+    },
+  },
+  setup() {
+    const { toggleMetaMaskSchema } = useConnectWallet();
+    const { currentAccount, currentAccountName } = useAccount();
+    const { width, screenSize } = useBreakpoints();
+    const iconWallet = ref<string>('');
+    const balUsd = ref<number>(0);
+    const isCheckingSignature = ref<boolean>(false);
+    const isLockdropAccount = ref<boolean>(false);
+    const { balance } = useBalance(currentAccount);
+    const { nativeTokenUsd } = usePrice();
+    const { requestSignature } = useMetamask();
+
+    const store = useStore();
+    const isDarkTheme = computed(() => store.getters['general/theme'] === 'DARK');
+    const substrateAccounts = computed(() => store.getters['general/substrateAccounts']);
+
+    const isH160 = computed(() => store.getters['general/isH160Formatted']);
+    const isEthWallet = computed(() => store.getters['general/isEthWallet']);
+    const currentNetworkIdx = computed(() => {
+      const chainInfo = store.getters['general/chainInfo'];
+      const chain = chainInfo ? chainInfo.chain : '';
+      return getProviderIndex(chain);
+    });
+    const blockscout = computed(
+      () =>
+        `${providerEndpoints[currentNetworkIdx.value].blockscout}/address/${currentAccount.value}`
+    );
+    const subScan = computed(
+      () => `${providerEndpoints[currentNetworkIdx.value].subscan}/account/${currentAccount.value}`
+    );
+
+    const copyAddress = async () => {
+      await navigator.clipboard.writeText(currentAccount.value);
+      store.dispatch('general/showAlertMsg', {
+        msg: 'Copy address success!!',
+        alertType: 'success',
+      });
+    };
+
+    watchEffect(() => {
+      const account = getSelectedAccount(substrateAccounts.value);
+      if (!currentAccount.value) {
+        // Memo: placeholder
+        iconWallet.value = supportWalletObj[SupportWallet.PolkadotJs].img;
```

Comment:
Instead of using the polkadot-js wallet icon? How about showing an identicon? https://github.com/polkadot-js/ui/tree/master/packages/vue-identicon
The polkadot-js is a wallet-specific logo.

cc @Kahonnohak

Reply:
@hoonsubin 
It's not only polkadot.js but it displays each wallet icon here. Do you suggest replacing the wallet icon with the logo provided in [this](https://github.com/polkadot-js/ui/tree/master/packages/vue-identicon) package?

![image](https://user-images.githubusercontent.com/92044428/159231574-b706871f-1ee2-4f6d-8358-de0a303ed83e.png)

Reply:
I personally think using the identicon is better since all Polkadot web wallets use it. Plus, this method will be difficult for Ethereum wallet logins because users can choose to login with one wallet and actually sign the transaction with another (ex, Trust vs MetaMask). But I think I'll leave the final decision to @Kahonnohak since I like either options.

Reply:
That something I wanted to discuss thanks, my personal guess is no one is actually care about their identicon either js nor Metamask but this is something we can consider later on. I would say to show the wallet icon that people can really understand what wallet(account) you are connecting with. I'm not fully understand what you mention about Trust vs Metamask, would be great if we can discuss this bits at Dev sync later.